translation/beyondtheveil/zh_cn.lang

```diff
@@ -0,0 +1,1110 @@
+interact.idol.fish_quest=我的心脏在颤抖。可怖之物已被言说，联结得以强化。
+interact.idol.drowned=我流泪了。我获得了新知，一个令人痛苦的真相。
+interact.lacrymatory.full=这个泪瓶已被绑定。
+interact.lacrymatory.success=%1$s现在已经与泪瓶绑定。
+interact.citymapper.loading=城市测绘台正在缓慢地加载周围环境。
+interact.citymapper.player0=其他人现在正在使用城市测绘台。
+interact.citymapper.player1=其他人仍在使用城市测绘台。
+interact.citymapper.player2=我仍无法使用城市测绘台...因为其他人在使用它。
+interact.citymapper.player3=有个蠢货还在使用城市测绘台，导致我没法用。
+interact.citymapper.player4=有人还在使用测绘台...我们应该赶走他们。
+interact.citymapper.player5=其他人现在正在使用城市测绘台。
+interact.heart.toofar=另一颗心脏距离太远了。
+interact.heart.notfound=另一颗心脏可能出了问题。我应该去检查一下。
+interact.heart.stored=这颗心脏的位置已经保存在我的珊瑚血杖中。
+interact.heart.printlink=这颗心脏连接的坐标为 x： %1$s， y： %2$s， z： %3$s。
+interact.heart.nolink=这颗心脏尚未连接。
+interact.heart.cleared=这颗心脏的连接已被清除。
+interact.heart.linked=位于坐标 x： %1$s， y： %2$s， z： %3$s 的心脏现在已经连接这颗心脏。
+interact.sacrificial_altar.incomplete=祭坛结构不完整。
+interact.blood_well.amount=这个血井中保存着 %1$s 个血骷髅和 %2$s 个血僵尸。确切来说，它们被保存在与此相连的异界之地。
+interact.statue.ownerelse=这个雕像为 %1$s 所有。
+interact.statue.ownerme=这个雕像是我的。
+interact.statue.noowner=这个雕像将为我所有。
+interact.statue.ownerboh=我不知道这个雕像联系着谁。
+interact.statue.alreadyme=这个雕像已经是我的了。
+interact.statue.alreadyelse=这个雕像已经与其他人绑定，其血契无法被打破。
+interact.statue.bound=这个雕像浴我之血。它已经与我绑定，我们的联系牢不可破，坚不可摧。
+interact.gearbench.dunno=我不知道如何使用这个装置。
+interact.lacrymatory.amount=这个泪瓶存有 %1$s/4000 毫桶的泪。
+
+hit.idol.schiz0=我不该试图破坏它。
+hit.idol.schiz1=我不该这么做。
+hit.idol.schiz2=我应该停下。
+hit.idol.schiz3=我真的该停手了。
+hit.idol.schiz4=停。
+hit.idol.schiz5=停止。
+hit.idol.schiz6=立刻停止。
+hit.idol.schiz7=你真的该收手了。
+hit.idol.schiz8=§5停下
+hit.idol.schiz9=你这个蠢货。
+hit.idol.schiz10=你根本不知道自己在做什么。
+
+##################
+#多方块结构
+##################
+
+multiblock.layer=层
+multiblock.blood_well=血井
+multiblock.sacrifice_altar=献祭祭坛
+multiblock.dream_shrine=降梦神龛
+
+multiblock.dream_shrine.checksout=降梦神龛搭建正确。
+multiblock.dream_shrine.noshrine=这不是降梦神龛的结构。
+multiblock.sacrifice_altar.checksout=献祭祭坛搭建正确。
+multiblock.sacrifice_altar.noaltar=这不是献祭祭坛的结构。
+
+##################
+#实体
+##################
+
+entity.deep_one.name=深潜者
+entity.hamlet_dweller.name=旧渔村居民
+entity.canoe.name=独木舟
+entity.crawling_villager.name=爬行的村民
+entity.weeper.name=泣者
+entity.fletum.name=泣囊
+entity.shoggoth.name=修格斯
+entity.crazed_weeper.name=发狂的泣者
+entity.blood_zombie.name=血僵尸
+entity.blood_skeleton.name=血骷髅
+entity.surgeon.name=外科医兽
+
+shoggoth.hasmapalready0=这个修格斯正在筑造建筑...打断这件事恐怕不是什么明智之举，对吧？
+shoggoth.hasmapalready1=当修格斯在做自己的事情时，我应该停止尝试指挥牠。这是很危险的一件事。
+shoggoth.hasmapalready2=只有蠢货才会尝试把修格斯的注意力转移到自己身上...我可没那么蠢，对吧？
+shoggoth.hasmapalready3=停下来。
+
+##################
+#GUI和按键
+##################
+
+gui.dialogue.talk=交谈
+gui.dialogue.trade=交易
+gui.sleep_chamber.wake=醒来
+
+gui.tablet.inscription0=There is nothing in the Void.<BR>虚空无物。<BR>And yet everything comes from There.<BR>然亦万物之源。<BR>That black portal to Infinity.<BR>邃暗门后，可见无限。
+gui.tablet.inscription1=Knowledge is to be feared,<BR>知识宜惧，<BR>not sought.<BR>莫当索求。<BR>It may bring Power,<BR>虽携神力，<BR>but at what cost?<BR>代价何由？
```

Comment:
```suggestion
gui.tablet.inscription1=Knowledge is to be feared,<BR>知识当惧，<BR>not sought.<BR>更莫索求。<BR>It may bring Power,<BR>虽携神力，<BR>but at what cost?<BR>代价何由？
```